package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radar-sdk-js",
-  "version": "3.0.0-beta.8",
+  "version": "3.0.1",
```

Comment:
ooh, nice catch. Maybe we should test for this too in `version.test.js`

Reply:
agreed, but still kinda feels odd to me to be "manually" bumping this value

Reply:
Yeah, I don't think we should be manually editing this file. But the test should ensure that `npm install` is run before committing (I assume that's how this field gets updated).

Reply:
right, but unfortunately `npm i` locally resulted in those other package version changes... that's why I ended up having to go manual

Reply:
Try nuking `node_modules` first? We also might need to add some of those packages to `package.json` to make sure they're installing the correct versions.

Reply:
tried doing so on this branch and on master without success - I added the following as dependencies but `npm i` was hellbent on using the old versions of `minimist` / `just-extend`, even going as far as installing those versions independently as subdependencies:
package.json
```
"just-extend": "^4.0.0",
"minimist": "^0.2.1",
```
package-lock.json diff after:
```
@@ -4906,7 +4906,15 @@
       "integrity": "sha1-MAV0OOrGz3+MR2fzhkjWaX11yQM=",
       "dev": true,
       "requires": {
-        "minimist": "0.2.1"
+        "minimist": "0.0.8"
+      },
+      "dependencies": {
+        "minimist": {
+          "version": "0.0.8",
+          "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
+          "integrity": "sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=",
+          "dev": true
+        }
       }
     },
     "mocha": {
@@ -5026,10 +5034,18 @@
       "dev": true,
       "requires": {
         "@sinonjs/formatio": "3.0.0",
-        "just-extend": "^4.0.0",
+        "just-extend": "^3.0.0",
         "lolex": "^2.3.2",
         "path-to-regexp": "^1.7.0",
         "text-encoding": "^0.6.4"
+      },
+      "dependencies": {
+        "just-extend": {
+          "version": "3.0.0",
+          "resolved": "https://registry.npmjs.org/just-extend/-/just-extend-3.0.0.tgz",
+          "integrity": "sha512-Fu3T6pKBuxjWT/p4DkqGHFRsysc8OauWr4ZRTY9dIx07Y9O0RkoR5jcv28aeD1vuAwhm3nLkDurwLXoALp4DpQ==",
+          "dev": true
+        }
       }
     },
```

Reply:
thinking about this a bit more - these are all `devDependencies`, not sure the version matters if the packages won't make it to the final build

Reply:
Yeah, these things are only for local development. Probably good to merge for now, can figure this out later.